packages/form-js-carbon-styles/src/carbon-styles.js

```diff
@@ -1047,27 +1047,48 @@ const CARBON_STYLES = css`
   ${SELECT_STYLES}
 
   .fjs-container {
+    width: 100%;
+    height: min-content;
+    --carbon-breakpoint-lg-width: 66rem;
+
     .cds--grid {
-      padding: 0 var(--cds-spacing-06);
+      padding: 0;
+    }
+
+    .fjs-form {
+      background-color: transparent;
+      color: var(--cds-text-primary);
+      padding: 0;
+    }
+
+    .fjs-layout-column:first-of-type .fjs-form-field {
+      margin-left: 0;
+    }
+
+    .fjs-layout-column:last-of-type .fjs-form-field {
+      margin-right: 0;
+    }
+
+    @media (width < var(--carbon-breakpoint-lg-width)) {
+      .fjs-layout-column .fjs-form-field {
+        margin-left: 0;
+        margin-right: 0;
+      }
+    }
+
+    .fjs-input-group {
+      margin: 0;
+    }
+
+    .fjs-powered-by,
+    .fjs-form-field button[type="submit"] {
+      display: none;
```

Comment:
Imho, we should not include this here. Once we use these styles in the form-playground, this will break as you can place it from the palette, but won't see them on the canvas.

Couldn't we keep this particular one in Tasklist?

Reply:
Yep, I agree. It was just a copy&paste mistake.